src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -109,20 +115,23 @@
        [:div {}
         (when (:show-access-level-select? @state)
           (let [checkbox (fn [label]
-                           [:div {}
-                            [:label {:style {:cursor "pointer"}}
-                             [:input {:type "checkbox"
-                                      :checked (get (:selected-types @state) label)
-                                      :onChange #(swap! state update-in [:selected-types label] not)
-                                      :style {:cursor "pointer"}}]
-                             [:span {:style {:marginLeft "0.5ex"}} label]]])]
+                           (let [stateval (get (:selected-types @state) label)]
+                             [:div {}
+                              [:label {:style {:cursor "pointer"}}
+                               [:input {:type "checkbox"
+                                        :checked (or (nil? stateval) stateval)
+                                        :onChange #(swap! state update-in [:selected-types label] false?)
```

Comment:
I explicitly changed this from `not` to `false?`. In the case where the user does not have any saved state for a given label, we default the checkbox to checked and enable the filter. However, when the user clicks the checkbox and `:onChange` fires, we would attempt to `(not nil)` which is `true`. End result is that it would take two clicks for the user to uncheck the checkbox. `false?` does the right thing here for the user.